.travis.yml

```diff
@@ -18,16 +18,23 @@ install:
   - conda config --set always_yes yes --set changeps1 no
   - conda update --quiet --yes conda
   - conda info -a
+  - conda config --add channels conda-forge
   - conda env create --quiet --name rbc-ci --file .conda/environment.yml
```

Comment:
rbc and omniscidb need likely different environments. So, I suggest to try the following approach:
```
conda env create --quiet --name rbc-ci --file .conda/environment.yml
conda env create --quiet --name omnisci-ci omniscidb
conda activate rbc-ci
...
```
and remove the `conda install omniscidb -c conda-forge` below.

Then:
```
before_script:
  - conda activate omnisci-ci
  - mkdir data && omnisci_initdb data -f
  - omnisci_server --enable-runtime-udf --enable-table-functions &
  - conda deactivate omnisci-ci
```